crates/holochain_sqlite/src/sql/cell/schema.sql

```diff
@@ -106,7 +106,8 @@ CREATE TABLE IF NOT EXISTS DhtOp (
     -- an INTEGER.
     when_integrated_ns  BLOB NULL,          -- DATETIME
 
-    receipt_count       INTEGER     NULL,
+    -- BOOLEAN the op has received enough validation receipts.
+    receipts_complete   INTEGER     NULL,
```

Comment:
```suggestion
    -- The op has received enough validation receipts.
    -- This is required as a field because different ops have different EntryTypes,
    -- which have different numbers of required validation receipts.
    receipts_complete   INTEGER     NULL,     -- BOOLEAN
```